PencilCase/PencilCase/scripts/Utils.ts

```diff
@@ -0,0 +1,8 @@
+﻿export function guid(): string {
```

Comment:
Used to generate guid string in javascript.
Since using module loading but not namespace. So no need to wrap this function into a class, instead of that, just put it into one file.